docs/release-notes/whats-new-3.rst

```diff
@@ -0,0 +1,64 @@
+.. py:currentmodule:: litestar
+
+
+What's changed in 3.0?
+======================
+
+This document is an overview of the changes between version **2.x** and **3.0**.
+For a detailed list of all changes, including changes between versions leading up to the
+3.0 release, consult the :doc:`/release-notes/changelog`.
+
+
+Imports
+-------
+
++----------------------------------------------------+------------------------------------------------------------------------+
+| ``2.x``                                            | ``3.x``                                                                |
++====================================================+========================================================================+
+| **OpenAPI**                                                                                                                 |
++----------------------------------------------------+------------------------------------------------------------------------+
+| ``litestar.openapi.OpenAPIController``             | :mod:`.openapi.plugins`                                                |
```

Comment:
not sure if this is a good idea or not but what if we add, in v3 branch only, an intersphinx link to Litestar v2 docs so was can directly link things like `litestar.openapi.OpenAPIController`? Idk if it would even work.. just a thought